src/topology/algebra/uniform_ring.lean

```diff
@@ -31,6 +32,11 @@ open_locale classical
 noncomputable theory
 universes u
 
+lemma uniform_continuous_mul_const (α : Type*) [ring α] [uniform_space α] [uniform_add_group α]
```

Comment:
```suggestion
lemma uniform_continuous_const_mul (α : Type*) [ring α] [uniform_space α] [uniform_add_group α]
```
`(*) r` is multiplication by a constant on the left, not the right.

Are you sure you can't prove this with `uniform_continuous_const_smul`?